mm2src/mm2_metrics/src/mm_metrics.rs

```diff
@@ -544,32 +535,4 @@ mod test {
 
         block_on(async { Timer::sleep(6.).await });
     }
-
-    #[test]
-    fn test_prometheus_format() {
```

Comment:
Could you please check manually if Prometheus works correctly with new metrics implementation?
Here is the documentation on [how to setup Prometheus](https://developers.komodoplatform.com/basic-docs/atomicdex/atomicdex-tutorials/atomicdex-metrics.html#prometheus-integration).

Reply:
It could be great if you also remain the `test_prometheus_format` test.
To get a valid metrics prometheus message (that can be used as an `expected` value), you can run add the test with the same metrics values to the `dev` branch and just print the result rendered message.
Then you can port the same test to your branch and add `assert_eq!(actual, expected)` at the end of the test.

Reply:
yes, Prometheus works correctly

Reply:
> It could be great if you also remain the `test_prometheus_format` test.
> To get a valid metrics prometheus message (that can be used as an `expected` value), you can run add the test with the same metrics values to the `dev` branch and just print the result rendered message.
> Then you can port the same test to your branch and add `assert_eq!(actual, expected)` at the end of the test.

I can do this but the `collect_prometheus_format` sometimes comes in randomised form so you can't really make a test for it.. running first test might pass while rerunning the test again fails

Reply:
Ok I see, thank you